config/captcha.php

```
@@ -0,0 +1,29 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Site Key for google's recaptcha 
+    |--------------------------------------------------------------------------
+    |
+    | Here you may provide the site key for google's recaptcha. 
+    | A default option is provided that is given for testing purposes by google.
+    |
+    */
+
+    'site_key' => env('RECAPTCHA_SITE_KEY', '6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI'),
```

No need a default. it is not secure. Let the code fail.

---

Fixed